src/app/(main)/my-voyage/[teamId]/sprints/components/SprintWrapper.tsx

```diff
@@ -20,22 +20,24 @@ import { Meeting, Sprint } from "@/store/features/sprint/sprintSlice";
 import { getCurrentSprint } from "@/utils/getCurrentSprint";
 import { AsyncActionResponse, handleAsync } from "@/utils/handleAsync";
 import { GET } from "@/utils/requests";
-import { CacheTag } from "@/utils/cacheTag";
+// import { CacheTag } from "@/utils/cacheTag";
 import { getAccessToken } from "@/utils/getCookie";
 import { getUser } from "@/utils/getUser";
 import { VoyageTeamMember } from "@/store/features/user/userSlice";
 
 async function fetchMeeting({
+  sprintNumber,
   meetingId,
 }: FetchMeetingProps): Promise<AsyncActionResponse<FetchMeetingResponse>> {
+  console.log("fetch meeting");
   const token = getAccessToken();
 
   const fetchMeetingAsync = () =>
     GET<FetchMeetingResponse>(
       `api/v1/voyages/sprints/meetings/${meetingId}`,
       token,
       "force-cache",
-      CacheTag.sprint,
+      `sprint-${sprintNumber}`,
```

Comment:
can you actually refactor the cache tags to be same as how route paths is setup. I didn't think we'd need something like this, which is why I chose to use Enums for them. You'll need to change all the code where cache tags is used too though

Reply:
Okay, I can write a helper function